airesources/Julia/hlt/game.jl

```diff
@@ -0,0 +1,89 @@
+using Memento
+
+logger = get_logger("bot-logger")
+set_level(logger, "debug")
+
+struct Game
+    id::String
+    botname::String
+    initial_game_map::GameMap
+    width::Int
+    height::Int
+
+    function Game(botname::String)
+        id = get_msg()
+        setup_logger(botname, id)
+        botname = @sprintf("%s-%s", botname, id)
+
+        width, height = map(x -> parse(Int, x), split(get_msg()))
+        send_msg(botname, true)
```

Comment:
Can you address https://github.com/HaliteChallenge/Halite-II/issues/151?

Reply:
Fixed.